homeassistant/components/manual_mqtt/alarm_control_panel.py

```diff
@@ -143,8 +155,18 @@ def _state_schema(state):
                 vol.Optional(STATE_ALARM_TRIGGERED, default={}): _state_schema(
                     STATE_ALARM_TRIGGERED
                 ),
-                vol.Required(mqtt.CONF_COMMAND_TOPIC): mqtt.valid_publish_topic,
-                vol.Required(mqtt.CONF_STATE_TOPIC): mqtt.valid_subscribe_topic,
+                vol.Optional(
+                    mqtt.CONF_COMMAND_TOPIC, default=DEFAULT_COMMAND_TOPIC
+                ): mqtt.valid_publish_topic,
+                vol.Optional(
```

Comment:
Please remove all changes except the ones that updates this integration with the changes in the manual alarm integration.

Reply:
@MartinHjelmare --
> Please remove all changes except the ones that updates this integration with the changes in the manual alarm integration.

Sure, I can separate this out into multiple PRs, provided I don't have to wait another 2 months for the split-out PR to get reviews.

Reply:
I'm not sure we will accept the other changes. That will need to be discussed first with the community in an architecture issue.